### PR TITLE
Merge in `feature-armhf-native`

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -55,7 +55,8 @@ variables:
   # Workaround for GitLab intermittent networking
   FF_NETWORK_PER_BUILD: 1
   # Builder image version
-  IMAGE_VERSION: "master"
+  # Use the images created in the mender-test-containers feature branch
+  IMAGE_VERSION: "feature-armhf-native"
```
Comment (lluiscampos) [inline .gitlab-ci.yml:59]: Note to self: this one needs an update after merging https://github.com/mendersoftware/mender-test-containers/pull/545

File: docker-build-package [hunk 2]
```diff
@@ -180,7 +187,8 @@ docker run --rm \
         --env MENDER_PRIVATE_REPO_ACCESS_USER \
         --env MENDER_PRIVATE_GPG_KEY_BUILD \
         --env OVERRIDE_DEBIAN_SUFFIX \
-        ${IMAGE_NAME_PREFIX}-${DISTRO}-${RELEASE}-${ARCH}-${IMAGE_VERSION:-master} \
+        ${PLATFORM} \
+        ${IMAGE_NAME_PREFIX}-${DISTRO}-${RELEASE}-${ARCH}-${IMAGE_VERSION:-feature-armhf-native} \
```
Comment (lluiscampos) [inline docker-build-package:191]: Note to self: remove this too

Comment (lluiscampos) [reply docker-build-package:191]: Done

**Comment by lluiscampos** [comment]
Depends on https://github.com/mendersoftware/mender-test-containers/pull/545

Same here, @kacf feel free to have a last look before merging to master :+1:

**Review by kacf — APPROVED** [review]

**Pull request merged**